### PR TITLE
Add spark etl job main class

### PR DESCRIPTION
**Pull request by wyb**
1. Add SparkEtlJob class
2. Remove DppResult comment
3. Support loading from hive table directly

#3433

### CODE REVIEW

File: fe/src/main/java/org/apache/doris/load/loadv2/etl/SparkEtlJob.java [hunk 1]
```diff
@@ -0,0 +1,241 @@
...
+        this.etlJobConfig = null;
+        this.hiveSourceTables = Sets.newHashSet();
+        this.tableToBitmapDictColumns = Maps.newHashMap();
+    }
+
+    private void initSparkEnvironment() {
+        spark = SparkSession.builder().enableHiveSupport().getOrCreate();
+    }
+
+    private void initHiveConfigs(Map<String, String> hiveProperties) {
```
Comment (wangbo) [inline fe/src/main/java/org/apache/doris/load/loadv2/etl/SparkEtlJob.java:73]:
```suggestion
    private void iniSparkConfigs(Map<String, String> sparkProperties) {
```

**Review by imay — DISMISSED** [review]
LGTM

File: fe/src/main/java/org/apache/doris/load/loadv2/BrokerLoadJob.java [hunk 2]
```diff
@@ -17,6 +17,8 @@
 
 package org.apache.doris.load.loadv2;
 
+import static org.apache.doris.common.DataQualityException.QUALITY_FAIL_MSG;
```
Comment (morningman) [inline fe/src/main/java/org/apache/doris/load/loadv2/BrokerLoadJob.java:20]: Do not import static

**Review by morningman — DISMISSED** [review]
LGTM

**Review by morningman — APPROVED** [review]
LGTM

**Pull request merged**